### ISSUE TITLE
Unable to disable constraints on a table with primary key references by a foreign key

### ISSUE THREAD

**Issue by tomsquest** [tomsquest/datability#3]
Given the table

```
CREATE TABLE myTable
(
  id INTEGER NOT NULL,
  CONSTRAINT myPk PRIMARY KEY (id),
  CONSTRAINT myFk FOREIGN KEY (id) REFERENCES myTable (id)
);
```

When calling `Databases.postgresql(conn).dropAll("myTable")`
Then the error `ERROR: cannot drop constraint myPk on table myTable because other objects depend on it` is returned

Analysis : using `cascade` does the job, eg. `alter table myTable drop CONSTRAINT myPk CASCADE` .

**Comment by tomsquest** [comment]
Fixed in bea98ecd1829df6f9da2069bb36c0e09a0d482e1

**Comment by tomsquest** [comment]
Fixed in bea98ecd1829df6f9da2069bb36c0e09a0d482e1
